### DISCUSSION TITLE
Is there any way to let pymupdf to use system fonts?

### DISCUSSION THREAD

**Discussion by hujunyao** [pymupdf/PyMuPDF#1882] [Q&A] [answered]
I got some pdfs which some fonts was not embeded. however , the fonts pdf used is in system fonts directory. Is there any ways to use system fonts to render the pdf?
I search web, cannot find the results , it seems pymupdf ignore the system fonts?
I donnot know if this is a bug or a requested feature or just a question?

**Comment by JorjMcKie** [comment]
moving this to "Discussions" first

**Comment by JorjMcKie** [comment, marked as answer]
Indeed, (Py-) MuPDF cannot support system fonts. This can not be changed.
All you can do is referring to the respective font files.
